client/modules/accounts/containers/passwordReset/forgotContainer.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component, PropTypes } from "react";
+import { Meteor } from "meteor/meteor";
+import { i18next } from "/client/api";
+import { composeWithTracker } from "/lib/api/compose";
+import { MessagesContainer } from "../helpers";
+import { Forgot } from "../../components";
+
+class ForgotContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      formMessages: props.formMessages,
+      isLoading: false,
+      isDisabled: false
+    };
+
+    this.handleFormSubmit = this.handleFormSubmit.bind(this);
+    this.formMessages = this.formMessages.bind(this);
+    this.hasError = this.hasError.bind(this);
+  }
+
+  handleFormSubmit = (event, email) => {
+    event.preventDefault();
+
+    this.setState({
+      isLoading: true
+    });
+
+    const emailAddress = email.trim();
+    const validatedEmail = LoginFormValidation.email(emailAddress);
+    const errors = {};
+
+    if (validatedEmail !== true) {
+      errors.email = validatedEmail;
+    }
+
+    if ($.isEmptyObject(errors) === false) {
```

Comment:
What is this? Is it jQuery? If so we don't want to be using any jQuery within React components.